Dockerfile

```diff
@@ -4,9 +4,23 @@ FROM  java:8
 
 MAINTAINER  Jordan Halterman <jordan.halterman@gmail.com>
 
+RUN apt-get update && apt-get install -y \
+  git \
+  stress
+
+ARG YOURKIT_VERSION=2017.02-b75
+ENV yourkit_version=$YOURKIT_VERSION
+RUN wget https://www.yourkit.com/download/YourKit-JavaProfiler-$yourkit_version.zip
+RUN unzip YourKit-JavaProfiler-$yourkit_version.zip
```

Comment:
Consider merging this into one RUN ... && \ ... command to have a single layer created for this group of operation that is logically related.